catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/CorbaUtils.java

```diff
@@ -79,7 +83,9 @@ public static String getNodeValue(Any any) {
                         value = date.toString();
                     }
                 }
-            } catch (org.omg.CORBA.MARSHAL | IllegalFieldValueException | BadKind e) {
+            } catch (BadKind e) {
+                //Ignore BadKind errors as this isn't a data parsing exception
+            } catch (org.omg.CORBA.MARSHAL | IllegalFieldValueException e) {
```

Comment:
Is it worth it to include the date in this logging statement?  That way we may get an idea of why it couldn't be parsed